canonical-websites/jp.ubuntu.com
# ReAdd talisker to the codebase

# Summary

Add talisker

# QA

- Add in `.env` `TALISKER_NETWORKS=172.17.0.1`
- add in `.env` `SENTRY_DSN=ASK_TOTO`
- `./run`
- Make sure website works as usual
- Go on `/_status/`
- Should have a list of actions
- `/_status/metrics`
- Should have prometheus metrics
- `/_status/test/sentry`
- Go https://sentry.is.canonical.com/canonical/jpubuntucom/
- Should see 2 test issues
